src/driver/driver.cpp

```diff
@@ -58,6 +58,8 @@ DriverStatus EvolutionDriver::Execute() {
     }
   } // END OF MAIN INTEGRATION LOOP ======================================================
 
+  pmesh->UserWorkAfterLoop(pinput, tm);
```

Comment:
It seems fine to reintroduce this for now, but @jmstone and I discussed turning ParthenonManager::PreDriver and ParthenonManager::PostDriver into something like Driver::PreExecute and Driver::PostExecute.  When that gets done, perhaps this functionality belongs there?

Reply:
Good point. I'll open an issue so that we can keep track of it.